### PR TITLE
cert: translation updates for 20.20.9

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: lang/da.js [hunk 1]
```diff
@@ -9,23 +9,23 @@ export default {
...
-	"components.form-element.input.text.tooShort": "",
-	"components.form-element.input.url.typeMismatch": "",
+	"components.form-element.input.email.typeMismatch": "E-mail er ikke gyldig",
+	"components.form-element.input.number.rangeOverflow": "Tal skal være lavere end {max}",
+	"components.form-element.input.number.rangeUnderflow": "Tal skal være højere end {min}",
+	"components.form-element.input.text.tooShort": "{label} skal være på mindst {minlength} tegn",
+	"components.form-element.input.url.typeMismatch": "URL-adresse er ikke gyldig",
 	"components.form-element.valueMissing": "{label} er påkrævet.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-error-summary.errorSummary": "Der blev fundet {count, plural, one { was 1 error } other { were { count } errors }} i den information, du sendte",
```
Comment (svanherk) [inline lang/da.js:18]: Translators missed some English here 😞

File: lang/de.js [hunk 2]
```diff
@@ -9,23 +9,23 @@ export default {
...
 	"components.input-date-time-range.endDate": "Enddatum",
-	"components.input-date-time-range.errorBadInput": "",
+	"components.input-date-time-range.errorBadInput": "{startLabel} muss vor {endLabel} liegen",
 	"components.input-date-time-range.startDate": "Startdatum",
 	"components.input-date-time.date": "Datum",
-	"components.input-date-time.errorMaxDateOnly": "",
-	"components.input-date-time.errorMinDateOnly": "",
-	"components.input-date-time.errorOutsideRange": "",
+	"components.input-date-time.errorMaxDateOnly": "Datum muss vor {MaxDate} liegen",
+	"components.input-date-time.errorMinDateOnly": "Datum muss nach {MinDate} liegen",
```
Comment (svanherk) [inline lang/de.js:27]: Should these two be lowercase `m`?

File: lang/da.js (hunk 1 above)
Comment (margaree) [reply lang/da.js:18]: Yeah :( Should I message Jeff about that?

Comment (svanherk) [reply lang/da.js:18]: Or you can use the slack workflow (in the #localization slack channel, click the lightning bolt and fill out the "Translation Error Report").  I _think_ that will flag it for them to re-request it, but you could check with Jeff to confirm

**Pull request merged**